### PR TITLE
Support baichuan2-chat chat template

### PR DESCRIPTION
**Pull request by wangruohui**
This PR is to support baichuan2-7b-chat in turbomind.

Checnklist:
- [x] Support chat template. (but we can not handle the <eos> from assistant)
- [x] Update supported model in readme.
- [x] Support NormHead during model conversion
- [x] FP16 aligned with hf model (except for the eos)
- [ ] W4A16

### CODE REVIEW

**Comment by lvhan028** [comment]
May update supported models in readme
Could you please test it again after quantizing  into w4a16 model?

**Comment by lvhan028** [comment]
@wangruohui we may handle weight normalization in deploy.py. @irexyc Could you help with that?

File: README.md [hunk 1]
```diff
@@ -55,11 +55,12 @@ LMDeploy is a toolkit for compressing, deploying, and serving LLM, developed by
...
-| :------: | :-------------: | :--: | :-----: | :---: | :--: |
-|  Llama   |       Yes       | Yes  |   Yes   |  Yes  |  No  |
-|  Llama2  |       Yes       | Yes  |   Yes   |  Yes  |  No  |
-| InternLM |       Yes       | Yes  |   Yes   |  Yes  |  No  |
+|  Models   | Tensor Parallel | FP16 | KV INT8 | W4A16 | W8A8 |
+| :-------: | :-------------: | :--: | :-----: | :---: | :--: |
+|   Llama   |       Yes       | Yes  |   Yes   |  Yes  |  No  |
+|  Llama2   |       Yes       | Yes  |   Yes   |  Yes  |  No  |
+| InternLM  |       Yes       | Yes  |   Yes   |  Yes  |  No  |
+| Baichuan2 |       No        | Yes  |   No    |  No   |  No  |
```
Comment (lvhan028) [inline README.md:63]: Doesn't  `python lmdeploy/turbomind/chat.py ./workspace --tp 2` work?
What's the error message?

Let's put `TBD` under `KV INT8` and `W4A16` .

Comment (wangruohui) [reply README.md:63]: It supported. Just not tested at the moment of previous commit.

**Comment by lvhan028** [comment]
May update news in README

**Review by lvhan028 — APPROVED** [review]

**Review by lzhangzz — APPROVED** [review]

**Pull request merged**